stable/rabbitmq/templates/configuration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
you are missing labels in this file:

```
labels:
    app: {{ template "rabbitmq.name" . }}
    chart: {{ template "rabbitmq.chart". }}
    release: "{{ .Release.Name }}"
    heritage: "{{ .Release.Service }}"
```

Reply:
I noticed that PVC has this also missing so I fixed both